<launch>
	<!-- Lidar odometry param -->
	<rosparam file="$(find lvi_sam)/config/KITTI_lidar.yaml" command="load" />
	<!-- VINS config -->
	<param name="vins_config_file" type="string" value="$(find lvi_sam)/config/KITTI_camera.yaml" />


    <!--- Robot State TF -->
    <include file="$(find lvi_sam)/launch/include/module_robot_state_publisher.launch" />
    <!--- Run Rviz-->
    <include file="$(find lvi_sam)/launch/include/module_rviz.launch" />
    <!--- SAM -->
    <include file="$(find lvi_sam)/launch/include/module_sam.launch" />
</launch>
